src/kemal/common_error_handler.cr

```diff
@@ -0,0 +1,17 @@
+
+class Kemal::CommonErrorHandler < HTTP::Handler
+  INSTANCE = new
+
+  def call(context)
+    begin
+      call_next context
+    rescue ex : Kemal::Exceptions::RouteNotFoundException
+      Kemal.config.logger.write("Exception: #{ex.to_s}: #{ex.message}\n")
+      return render_404(context)
+    rescue ex
+      Kemal.config.logger.write("Exception: #{ex.to_s}: #{ex.message}\n")
+      return render_500(context, ex.to_s)
+    end
+  end
+
+end
```

Comment:
Was like this before:

    def process_request(context)
      url = context.request.path.not_nil!
      Kemal::Route.check_for_method_override!(context.request)
      lookup = @tree.find radix_path(context.request.override_method as String, context.request.path)
      if lookup.found?
        route = lookup.payload as Route
        context.request.url_params = lookup.params
        begin
          body = route.handler.call(context).to_s
          context.response.print body
          return context
        rescue ex
          return render_500(context, ex.to_s)
        end
      end
      return render_404(context)
    end

How would you do it ?